services/netlify/netlify.tester.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const { isColorMatcher } = require('../color-matcher')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('netlify (valid)')
+  .get('/0028c6a7-b7ae-49f6-b847-917b40b5b13f')
```

Comment:
Make sure to include the `.json` extension on these tests

Reply:
Has been added, still looks like something's wrong.